controllers/controllers_suite_test.go

```diff
@@ -48,12 +48,13 @@ import (
 const defaultNamespace = "default"
 
 var (
-	k8sClient    client.Client
-	k8sManager   manager.Manager
-	testEnv      *envtest.Environment
-	ctx          context.Context
-	cancel       context.CancelFunc
-	fakeRecorder *record.FakeRecorder
+	k8sClient             client.Client
+	k8sManager            manager.Manager
+	testEnv               *envtest.Environment
+	ctx                   context.Context
+	cancel                context.CancelFunc
+	fakeReconcileRecorder *record.FakeRecorder
+	fakeExecRecorder      *record.FakeRecorder
```

Comment:
I think it's fine to use the same recorder in the test code, it'll simplify the code significantly and I don't think there is a major trade-off as far as test quality.

Reply:
Indeed. I was not sure what is better